src/main/kotlin/no/nav/familie/ef/sak/vilkår/regler/vilkår/AlderPåBarnRegel.kt

```diff
@@ -66,9 +65,7 @@ class AlderPåBarnRegel :
         val fødselsdato =
             metadata.barn
                 .firstOrNull { it.id == barnId }
-                ?.personIdent
-                ?.let { Fødselsnummer(it).fødselsdato }
-                ?: error("Finner ikke ident til barn=$barnId")
+                ?.fødselTermindato ?: error("Finnes ingen fødsel-termindato for barn=$barnId")
```

Comment:
usikker på om dette holder ?

Reply:
Tenker at det er dumt å bruke et felt som kun settes 60% av tiden...

har verdi her:105590
har ikke verdi her: 62047

Reply:
Det kan hende vi BURDE hatt inn fødselsdato som et felt i behandlingBarn basert på alt det vi har gjort frem til nå for å jobbe oss rundt problemet. Alternativet er at `HovedregelMetadata` inneholder barnMedSamvær og følgelig kan vi finne pdl-fødselsdato derfra og hente termindato dersom det ikke finnes registerdata for barnet.

Reply:
`hovedregelmetadata.vilkårgrunnlagDto.barnMedSamvær` bør brukes her, og finnes ikke den så kan vi bruke fødseltermindato